alternator/executor.cc

```diff
@@ -664,12 +664,48 @@ static future<bool> is_view_built(
 
 }
 
+void executor::notify_all_shards_of_newly_calculated_table_size(schema_ptr schema, std::uint64_t size_in_bytes, std::chrono::nanoseconds ttl) {
+    auto &table = schema->table();
+    auto &describe_table_info = _describe_table_info_for_tables[&table];
+    auto now = describe_table_info.size_in_bytes.set_now(size_in_bytes, ttl);
+
+    (void)container().invoke_on_others(
```

Comment:
Why are you not waiting for the returned future and just ignoring it with void()?
If you ignore a future like that, you'll probably get warnings logged that you ignored a future. I don't think that's the right thing to do.

(I see you have a big comment about this below, but it still feels to me like this might not be the right thing to do, and also that Seastar probably warns when the future resolves when it is outright ignored).